testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml

```diff
@@ -124,7 +124,19 @@ spec:
                 - name: ANSIBLE_GATHERING
                   value: explicit
                 image: quay.io/example/memcached-operator:v0.0.1
+                livenessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 6789
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
                 name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 6789
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
```

Comment:
livenessProbe using /readyz endpoint and readinessProbe using /healthz endpoint. Should be the other way around I guess.